xml/admin_gui_dashboard.xml

```diff
@@ -613,9 +626,9 @@
   <sect2 xml:id="dashboard-adding-roles">
    <title>Adding Custom Roles</title>
    <para>
-    Click <guimenu>Add</guimenu> in the top left of the table heading to add a
-    new custom role. Enter its name, description, and set required permissions
-    for individual topics.
+    Click <guimenu>Create</guimenu> in the top left of the table heading to add a
+    new custom role. Enter its <guimenu>Name</guimenu>, <guimenu>Description</guimenu>,
+    select the <guimenu>Permissions</guimenu>.
```

Comment:
```suggestion
    and some appropriate <guimenu>Permissions</guimenu>.
```

Reply:
You really do select appropriate permissions. It looks as below. If you can think of a better way to indicate that?
![Screenshot from 2020-08-19 13-00-38](https://user-images.githubusercontent.com/1987994/90632358-2cfd9780-e21c-11ea-93b6-9fc48b28f98b.png)

Reply:
So... since I am subscribed to this thread already, have my unasked-for comments:

* I'd suggest making this a new sentence like `Next to <guimenu>Permissions</guimenu>, enable appropriate permissions.`
* Also, I'd really suggest _avoiding_ filler words like "some".

Reply:
- Sounds good to me
- Agreed

Reply:
OK, but I still suggest that the 3rd instruction should be prefaced with "and" or "then":

e.g.
"Click 'Create', enter the 'Name' and 'Description', and select the permissions."